index.js

```diff
@@ -1,6 +1,6 @@
 const readline = require('readline');
 const fetch = require('node-fetch');
-const opn = require('opn');
+const open = require('open');
 
```

Comment:
there are some upstream changes that are causing a conflict with this. everything else looks good, so once these conflicts are resolved in this file, it'll be good to merge. thanks!